packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -25,6 +25,10 @@ export class OpGroupingManager {
 		}
 
 		for (const message of batch.content) {
+			// Blob attaches cannot be grouped (grouped batching would hide metadata)
+			if (message.deserializedContent.type === ContainerMessageType.BlobAttach) {
```

Comment:
I assume that's temporary work around, as this means we have a chance to run into op size limitations with this change, right? If so, would be great to had an ADO item tracking next steps and mention it in the code.

Reply:
Yes, it's a temporary workaround and batches with this ops can hit the size limitations.
[AB#4221](https://dev.azure.com/fluidframework/internal/_workitems/edit/4221)